init-tools.msbuild

```diff
@@ -3,9 +3,23 @@
     <TargetFramework>netcoreapp1.0</TargetFramework>
     <EnableDefaultItems>false</EnableDefaultItems>
     <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
-    <BaseIntermediateOutputPath>$(MSBuildThisFileDirectory)Tools/$(BuildToolsPackageVersion)</BaseIntermediateOutputPath>
+    <BaseIntermediateOutputPath>$(ToolsDir)/$(BuildToolsPackageVersion)</BaseIntermediateOutputPath>
+    <FeedTasksVersion>1.0.0-prerelease-02111-01</FeedTasksVersion>
```

Comment:
What's the strategy for keeping this version updated?

Reply:
Currently it's just checked in. Is it better to make it similar to how BuildToolsVersion.txt works?

Reply:
My experience is that having hard-coded version numbers that have no strategy for getting updated regularly get out of date/sync really quick.  And then when someone does need to update them, there are a bunch of breaking changes they need to work through.

I think it would be better if we had an automated approach to updating this version, just like we do with other dependencies.

Reply:
If there was a small buildtools package with a props file which defined the packages / versions that buildtools produced, then init-tools.cmd/sh could restore that project directly and init-tools.msbuild could use the props file to determine the set of packages to restore.

Reply:
We can definitely add an updater to dependencies.props to get it auto-updated which is what I would expect.

Reply:
This has now been updated.